isobus/src/isobus_virtual_terminal_server_managed_working_set.cpp

```diff
@@ -45,12 +47,15 @@ namespace isobus
 
 	VirtualTerminalServerManagedWorkingSet::~VirtualTerminalServerManagedWorkingSet()
 	{
-		for (auto key = vtObjectTree->begin(); key != vtObjectTree->end(); ++key)
+		if (nullptr != vtObjectTree)
 		{
-			if (nullptr != (*key).second)
+			for (auto key = vtObjectTree->begin(); key != vtObjectTree->end(); ++key)
 			{
-				delete (*key).second;
-				(*key).second = nullptr;
+				if (nullptr != (*key).second)
+				{
+					delete (*key).second;
```

Comment:
At some point we will probably want to change the raw pointer being used for the working set to be some kind of smart pointer, but again it's a low priority and doesn't matter too much right now since you're just merging back to a development branch.